index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Html Form</title>
+    <link rel="stylesheet" href="assets/css/styles.css"/>
+  </head>
+  <body>
+    <div class="top_right">
+      <div class="flex_container">
+        <p>Already have an Intuit account?</p>
+        <a href="https://accounts.intuit.com/account-recovery.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS">I forgot my User ID or Password</a>
+      </div>
+      <button class="button" type="button" name="button"><img src="assets/img/sign_in.png" alt="Not found"/></button>
+    </div>
+
+    <main class="center">
+      <div class="width">
+        <figure class="center">
+          <img src="assets/img/header.png" alt="Not found"/>
+        </figure>
+
+<!-- form -->
+        <form class="form" method="POST">
+          <div class="center_form">
+            <div class="center_text">
+              <h2>Create an Intuit account</h2>
+              <div class="center_text">
+                <h3>One account for everything Intuit, including Mint. </h3>
+                <a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account">Learn More.</a>
+              </div>
+            </div>
+
+            <div>
+
+
+              <div class="center_inputs">
+                <label>Email Adress</label>
+                <input type="text" placeholder="example@test.com"/>
+              </div>
+
+              <div class="center_inputs">
+                <label>Phone (recommended)</label>
+                <input type="tel" placeholder="(123) 4567890"/>
+                <p id="standard_call">Standard call, messaging or data rates may apply.</p>
+              </div>
+
+              <div class="center_inputs">
+                <label>Password</label>
+                <input type="password" placeholder="Enter your password"/>
```

Comment:
`SUGGESTION`

Check comment above on associating `<label>` and `<input>` elements.